### PR TITLE
Inlinekeyboard for setting global template

### PR DESCRIPTION
**Pull request by athulkrishnaaei**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
 /target
+.env
```
Comment (pxp9) [inline .gitignore:2]: Why this change ? 

Just don't push your tokens to your fork

File: src/bot/commands.rs [hunk 2]
```diff
@@ -66,15 +71,23 @@ pub trait Command {
...
 
     fn execute(&self, db_pool: Pool<ConnectionManager<PgConnection>>, api: Api, message: Message) {
-        info!(
-            "{:?} wrote: {}",
-            message.chat.id,
-            message.text.as_ref().unwrap()
-        );
+        let messages = message.text.as_ref().unwrap().to_string();
+
+        let response_message = &messages.replace(BOT_NAME, "");
```
Comment (pxp9) [inline src/bot/commands.rs:76]: Not understand this line of code, why you would like to delete bot name from a message?

Comment (athulkrishnaaei) [reply src/bot/commands.rs:76]: when we use inline keyboard bot name is added automatically in the text field,so if we send that to the bot commands will not work properly

File: .gitignore (hunk 1 above)
Comment (pxp9) [reply .gitignore]: Fixed in https://github.com/ayrat555/el_monitorro/pull/255/commits/1c44008e20d3220ef61e44e9a7393cb56758d831

File: src/bot/commands.rs (hunk 2 above)
Comment (pxp9) [reply src/bot/commands.rs:76]: Ok
